util/test/test_otel.py

```
@@ -0,0 +1,61 @@
+            "OTEL_EXPORTER_OTLP_PROTOCOL": "http",
+            "OTEL_TRACES_SAMPLER_ARG": 0.005,
+        },
+        "FEATURE_ENABLE_OTEL": True,
+    },
+]
+
+
+@pytest.fixture()
+def test_otel_config():
+    # since we can set tracer_provider only once, we go with the default one
+    app.config["OTEL_CONFIG"] = app_configs[0]["OTEL_CONFIG"]
```

Ruff (0.14.4)

Undefined name `app`

(F821)